src/test/java/io/vertx/test/core/FileDescriptorLeak.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2011-2019 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.test.core;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({java.lang.annotation.ElementType.METHOD})
+public @interface FileDescriptorLeak {
```

Comment:
What is this interface useful for ?

Reply:
It is added on a method that we need to check for leaks.

Without this and only the rule it would run in all the tests of each file (I think)

Reply:
In the PR you can see it added on the 

testSendFile in order to check for leaks both for http1 and http2

The same can be added on a net test / file system test and will instantly check for leaks

Reply:
yes I missed it

Reply:
Can you add a same test for other usage of send file in NetSocket and HTTP/1 ?

Reply:
Since we annotated the sendFile of HttpTests class then it works for both http1 and http2

You can clearly see that http1 passes and http2 fails

Reply:
indeed, good job

Reply:
can you rename it `DetectFileDescriptorLeak`